src/Theme/FooterTemplate.elm

```diff
@@ -54,7 +54,7 @@ translatedLogoPath language fileName =
             "/images/" ++ fileName ++ "-cy.svg"
 
 
-navigationColumn : { title : Key, links : List { text : Key, href : Key } } -> Language -> Route -> Html msg
+navigationColumn : { title : Key, links : List { text : Key, href : String } } -> Language -> Route -> Html msg
```

Comment:
I am worried that by using plain strings we may introduce un-catchable errors with the links. 

This is okay for now but maybe there could be some kind of formal "link" database we can key into so at least all the strings are in one place.

Reply:
I agree with what you're saying, but the links have to be hardcoded somewhere and I'm not sure that adding another file for this necessarily stops this from being a problem - it just shifts the problem elsewhere. I did try specifying them all as separate functions but it felt very OTT, so this navigation column function is in effect now just a type of data structure. Definitely something to consider in the future.

Reply:
i think it's more- if you're doing something potentially hacky then at least hide it in one place with a nice interface?

but yeah- not an issue at this time